<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentForm</class>
 <widget class="QWidget" name="AppointmentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schedule Appointment</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Dark theme styling */
QWidget {
    background-color: #2D2D2D;
    color: #E0E0E0;
    font-family: 'Segoe UI', Arial;
    font-size: 11pt;
}

QLabel {
    color: #E0E0E0;
    font-size: 12pt;
}

QGroupBox {
    border: 1px solid #444;
    border-radius: 8px;
    margin-top: 10px;
    padding-top: 15px;
    background-color: #3A3A3A;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px;
    color: #E0E0E0;
}

QPushButton {
    background-color: #3A3A3A;
    color: #E0E0E0;
    border: 1px solid #444;
    border-radius: 4px;
    padding: 8px 15px;
    min-width: 120px;
    font-size: 12pt;
}

QPushButton:hover {
    background-color: #4A4A4A;
}

QPushButton:pressed {
    background-color: #2A2A2A;
}

QPushButton:disabled {
    background-color: #2A2A2A;
    color: #777;
}

QComboBox, QDateEdit, QTextEdit {
    background-color: #3A3A3A;
    color: #E0E0E0;
    border: 1px solid #444;
    padding: 5px;
    border-radius: 4px;
    min-height: 25px;
}

QComboBox QAbstractItemView {
    background-color: #3A3A3A;
    color: #E0E0E0;
    selection-background-color: #0078D7;
}

QTextEdit {
    font-family: 'Segoe UI', Arial;
    font-size: 11pt;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Schedule Appointment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="reasonTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>760</width>
     <height>300</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Describe the reason for your appointment in detail</string>
   </property>
   <property name="placeholderText">
    <string>Enter reason for visit...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>520</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save the appointment details</string>
   </property>
   <property name="text">
    <string>Save Appointment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>520</y>
     <width>152</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Close without saving</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="formWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>760</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="patientComboBox">
      <property name="toolTip">
       <string>Select a patient from the list</string>
      </property>
      <property name="placeholderText">
       <string>Select Patient</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="doctorComboBox">
      <property name="toolTip">
       <string>Select a doctor from the list</string>
      </property>
      <property name="placeholderText">
       <string>Select Doctor</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QDateEdit" name="dateEdit">
      <property name="toolTip">
       <string>Select appointment date</string>
      </property>
      <property name="minimumDate">
       <date>
        <year>2023</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2023</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QComboBox" name="timeSlotComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Time slots will appear after selecting doctor and date</string>
      </property>
      <property name="placeholderText">
       <string>Select Time Slot</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
